### PR TITLE
Made footer fixed width [Fixes #23]

### LINKED ISSUE #23: fixed width container
if we're gonna do a fixed width container, the nav should be inside it as well 

http://screencast.com/t/QXGalKhHz

### PR DESCRIPTION
**Pull request by avophile**
Fixed-width the footer and made other small changes.

### CODE REVIEW

File: views/guide.lavender [hunk 1]
```diff
@@ -392,7 +394,8 @@ block content
         pre
           code.language-jade.
             p
```
Comment (mowchan) [inline views/guide.lavender:396]: `p` inside of `code`?

File: views/guide.lavender [hunk 2]
```diff
@@ -392,7 +394,8 @@ block content
...
-            a(href=&#39;/some/place&#39;)- click here for some cool stuff.
+              a(href=&#39;/some/place&#39;)- click here for some cool stuff.
+              | |.
```
Comment (mowchan) [inline views/guide.lavender:398]: Double `|`?

File: views/guide.lavender (hunk 1 above)
Comment (avophile) [inline views/guide.lavender:396]: @mowchan Yes to match the wiki: 

pipe on wiki

double pipe on page

File: views/guide.lavender (hunk 2 above)
Comment (avophile) [inline views/guide.lavender:398]: @mowchan Yes, see above.

File: views/guide.lavender (hunk 1 above)
Comment (mowchan) [inline views/guide.lavender:396]: Ah, :ok:

File: views/guide.lavender (hunk 2 above)
Comment (mowchan) [inline views/guide.lavender:398]: This is to match the wiki as well I assume?

Comment (mowchan) [inline views/guide.lavender:398]: :ok:

**Comment by mowchan** [comment]
:cake:

File: views/guide.lavender (hunk 2 above)
Comment (TomWoodward) [inline views/guide.lavender:398]: yea it just looks weird because we're rendering plaintext lavender with lavender

**Comment by TomWoodward** [comment]
:cake:

**Pull request merged**